src/modules/UniversalEmailRecoveryModule.sol

```diff
@@ -229,6 +229,15 @@ contract UniversalEmailRecoveryModule is
         return getGuardianConfig(account).threshold != 0;
     }
 
+    /**
+     * Check if the recovery module is authorized to recover the account
+     * @param account The smart account to check
+     * @return true if the module is authorized, false otherwise
+     */
+    function isAuthorizedToBeRecovered(address account) external view returns (bool) {
+        return validatorCount[account] > 0;
```

Comment:
Should I use `return authorized[account]` as same as the src/modules/EmailRecoveryModule.sol ?

Reply:
See comment on modified solution. Let me know what you think